plugins/modules/mso_schema_template.py

```diff
@@ -156,6 +168,11 @@ def main():
     else:
         schema_path = 'schemas'
 
+    if schema_description is None:
+        schema_description = ''
+    if template_description is None:
+        template_description = ''
```

Comment:
If doing this why not use a default = '' ?
Also is that creating any issues on previous versions?